### DISCUSSION TITLE
How many characters does ch_ptocr_v2_rec_infer model support?

### DISCUSSION THREAD

**Discussion by memofemanon** [PaddlePaddle/PaddleOCR#6084] [Q&A] [answered]
Hi,
I am checking how many Chinese characters ch_ptocr_v2_rec_infer supports and what are they. I noticed that the final layer of the model has **6625** channels, while there are only **6623** characters in the dictionary file ppocr_keys_v1.txt.
If I consider the blank, there is still 1 class missing, what is that?

**Comment by tink2123** [comment, marked as answer]
Yes ppocr_keys_v1.txt supports 6623 characters， the final layer of the model has 6625 channels because of the [CTC](https://en.wikipedia.org/wiki/Connectionist_temporal_classification) needs an extra meaningless link characters " - ".
